### ISSUE TITLE
Using batch normalization leads to issues

### ISSUE THREAD

**Issue by pat-alt** [JuliaTrustworthyAI/CounterfactualExplanations.jl#110]
In the past using `BatchNorm` inside Flux models worked fine, but this is now broken. Note exactly sure what's caused this, potentially the fact that data is now standardized or changes on Flux 's side.
